src/App/Fossa/Analyze.hs

```diff
@@ -147,7 +150,7 @@ analyzeMain workdir recordMode logSeverity destination project unpackArchives en
             let effects = analyzeEffects journal
             runReplay @ReadFS (effectsReadFS effects)
               . runReplay @Exec (effectsExec effects)
-              $ analyze basedir destination project unpackArchives enableVSI filters
+              $ analyze basedir destination project unpackArchives printMetadata enableVSI filters
```

Comment:
[nit]: This analyze call should be bound in a `where` clause so we don't have to repeat the changes three times.
```suggestion
              $ doAnalyze
  where
    doAnalyze = analyze basedir destination project unpackArchives printMetadata enableVSI filters
```